relay-dynamic-config/src/feature.rs

```diff
@@ -9,6 +9,9 @@ pub enum Feature {
     SessionReplay,
     /// Enables data scrubbing of replay recording payloads.
     SessionReplayRecordingScrubbing,
+    /// Enables combining session replay envelope item (Replay Recordings and Replay Events).
+    /// into one item.
+    SessionReplayCombinedEnvelopeItems,
```

Comment:
Is it currently possible to send non-combined events in different envelopes? How should relay handle that when this feature flag is enabled?

Reply:
Envelopes will always arrive in an uncombined state.  These combined envelopes are forwarded to our Kafka consumer.  If for whatever reason Relay can't or chooses not to combine the envelopes the consumer process will know how to handle the uncombined envelopes.

Reply:
ReplayEvents and ReplayRecordings will always be in the same envelope, and this has always been the case.

Reply:
> Envelopes will always arrive in an uncombined state.

@cmanallen If I understand the RFC correctly, SDKs will at some point send combined replay items though, right?

Reply:
@jjbayer 

> SDKs will at some point send combined replay items

I believe that requirement has been dropped.  I'll let @JoshFerge clarify the point.  To my knowledge, the SDK will not be updated to use a combined format.

---

Also I may have misunderstood the above statements so I'll clarify quickly.

>  ...send non-combined events in different envelopes

As in one request per each envelope item?  @JoshFerge is correct.  We do not support this.  What we support is a single HTTP request with two distinct envelope items.

Reply:
I'll update the RFC diagrams today to reflect this decision.